### ISSUE TITLE
Bump supported version

### ISSUE THREAD

**Issue by Nick6489** [pumper42nickel/eloquence_threshold#30]
Can someone please bump the supported NVDA version before 2023.1 is released? I'd do it myself but it turns out I can't build the add-on due to some connection not being able to be made.

**Comment by alexstine** [comment]
Upvote to this. I will give it a try if I can get my local env working.

**Comment by pumper42nickel** [comment]
PR would be welcome!

**Comment by Elshara** [comment]
I two am interested in this happening.

**Comment by alexstine** [comment]
@pumper42nickel I opened a PR but cannot test it because I do not have the original add-on. The URL in build.py is now invalid.

Hope it helps. Thanks.

**Comment by alexstine** [comment]
Finally figured out how to test this. Posted instructions in the PR. If you understand tech, should be easy enough. There is likely a better way but NVDA dev docs confuse me so I found my own work-around. Now running NVDA 2023.1 with no issues.

**Comment by Elshara** [comment]
Works! Thank you so much!
